src/Model/ModelData.php

```diff
@@ -677,4 +542,14 @@ public function Debug(): ModelData|string
     {
         return ModelDataDebugger::create($this);
     }
+
+    /**
+     * Generate the cache name for a field
+     */
+    private function objCacheName(string $fieldName, array $arguments = []): string
+    {
+        return empty($arguments)
+            ? $fieldName
+            : $fieldName . ":" . var_export($arguments, true);
```

Comment:
Use something that's not var_export() as this will cause the cache name to be split across multiple lines

Reply:
This code was already there and has proven to work well. I don't intend to change it unless you have a specific suggestion that will achieve what this is already achieving.

Reply:
If it was already there then it should be fine. I'd prefer md5() around it if it's low-risk to add though.

Reply:
Can do.